Dockerfile.ako-operator

```diff
@@ -24,6 +24,7 @@ LABEL version="v1.12.3"
 LABEL release="1"
 LABEL description="Manage configmap, statefulset and other artifacts for deploying AKO controller"
 LABEL vendor="VMware"
+LABEL maintainer="VMware"
```

Comment:
should it be Broadcom?

Reply:
@arihantg For this release, I intend to keep it as VMWare. The account settings still has VMWare everywhere. Also, this will require changes at all the places including logo etc. I am planning to do it all in the 1.13.2 release.